scss/modules/custom.scss

```diff
@@ -0,0 +1,6 @@
+.card__preview {
+  background-image: url(@images/background.jpg);
```

Comment:
This is the example how to specify source image file in SCSS using Webpack alias.
Webpack aliases are defined in the Webpack config to avoid using a relative path like `../../images`.